src/renderer/views/Playlist/Playlist.js

```diff
@@ -678,6 +687,13 @@ export default defineComponent({
         throw failure
       }
     },
+    formatDuration(totalSeconds) {
+      const hours = Math.floor(totalSeconds / 3600)
+      const minutes = Math.floor((totalSeconds % 3600) / 60)
+      const seconds = totalSeconds % 60
+
+      return `${hours}h ${minutes}m ${seconds}s`
```

Comment:
Something like this should be used instead so we get proper localization support:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DurationFormat#using_intl.durationformat